modules/mod_signup/mod_signup.erl

```diff
@@ -220,10 +220,12 @@ ensure_identity(Id, {Type, Key, IsUnique, IsVerified}, Context) when is_binary(K
 
 props_to_rsc(Props, IsVerified, Context) ->
     Category = z_convert:to_atom(m_config:get_value(mod_signup, member_category, person, Context)),
+    ContentGroup = z_convert:to_atom(m_config:get_value(mod_signup, content_group, default_content_group, Context)),
```

Comment:
I propose to have `undefined` as the default content group. As that works better in situations where `mod_content_groups` is not enabled.  And `default_content_group` is the default for 'undefined' content groups anyway.